dask/array/random.py

```diff
@@ -349,13 +349,9 @@ def zipf(self, a, size=None, chunks=None):
                           size=size, chunks=chunks)
 
 
-def _apply_random(func, seed, size, args, kwargs):
-    """ Apply RandomState method with seed
-
-    >>> _apply_random('normal', 123, 3, (10, 1.0), {})
```

Comment:
The example uses a seed, where we pass in a vector now. The example still would work I suppose, but provides slightly incorrect information.